tests/migrations/examples/0001_sql_migration.py

```diff
@@ -0,0 +1,19 @@
+from typing import Sequence
+
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_local(self) -> Sequence[operations.SqlOperation]:
+        return []
```

Comment:
This will work for any tests that don't need to check the operations. The operations depend on specifying a `StorageSetKey` so I wasn't sure how we envision a migrations module that is independent of snuba  `StorageSetKey`s

Reply:
We should have at least one operation in here to make this migration more realistic. Eventually storage set keys will probably need to get registered in the migration framework (and not just shared from Snuba), but for now I think it's ok to hardcode an operation using some existing Snuba storage set keys here. We can swap it out later.